src/FlightDisplay/FlyViewToolStripActionList.qml

```diff
@@ -66,7 +66,7 @@ ToolStripActionList {
             enabled: false
             visible: enabled
             onTriggered:{
-                viewer3DWindow.settingMenuOpen = !viewer3DWindow.settingMenuOpen
+                viewer3DWindow.settingsDialogOpen = !viewer3DWindow.settingsDialogOpen
```

Comment:
Triggering things other than visibility to happen on boolean state changes is pretty odd. This sort of thing is much better handled by a function in Viewer3DWindow like showSettingsDialog() which then creates the settings dialog component. Much clearer as to what it does.